src/TLuaInterpreter.cpp

```diff
@@ -10992,7 +10992,7 @@ void TLuaInterpreter::parseJSON(QString& key, const QString& string_data, const
         qDebug() << "CRITICAL ERROR: json_to_value not defined";
         return;
     }
-    lua_pushlstring(L, string_data.toLatin1().data(), string_data.size());
+    lua_pushlstring(L, string_data.toUtf8().constData(), string_data.size());
```

Comment:
Why do we have to use `lua_pushlstring` which requires a `size_t` argument compared to a `lua_pushstring` which doesn't but requires/uses an ASCII `nul` to terminate the string whereas the first can handle strings with embedded `nul`s - or is that _why_ it has to be used here...?

Reply:
No idea.